
# HG changeset patch
# User Daniel Atallah <datallah@users.sf.net>
# Date 1313371632 14400
# Node ID 3a74f971c6875f88649c8f0fcbaba8fbb8bf1d11
# Parent  ade24f79936d8bdf798ee31eb7682efec4178932
Use "dev" as the unreleased version suffix instead of "cvs".

diff -r ade24f79936d -r 3a74f971c687 configure.ac
--- a/configure.ac	Sun Aug 14 21:26:28 2011 -0400
+++ b/configure.ac	Sun Aug 14 21:27:12 2011 -0400
@@ -1,8 +1,8 @@
 dnl Process this file with autoconf to produce a configure script.
 
 AC_PREREQ(2.57)
-AC_INIT([gtkspell],[2.0.16+cvs],[gtkspell-devel@lists.sourceforge.net])
-dnl N.B. After 0.40, please change to `0.40+cvs' instead of `0.41cvs'.
+AC_INIT([gtkspell],[2.0.16+dev],[gtkspell-devel@lists.sourceforge.net])
+dnl N.B. After 0.40, please change to `0.40+dev' instead of `0.41dev'.
 dnl Rationale: (i) placate simple version comparison software such as
 dnl `dpkg --compare-versions'.  (ii) We don't always know what the next
 dnl version is going to be called until about the time we release it
@@ -32,7 +32,6 @@
 #   [AC_MSG_WARN([Enabling word-breaking workarounds since pango < 1.8.0])])
 
 # gtk-doc check:
-# gtk-doc check:
 # from http://cvs.gnome.org/lxr/source/gtk-doc/examples/configure.in
 AC_ARG_WITH(html-dir, AC_HELP_STRING([--with-html-dir=PATH],[path to installed docs]))
 

